Header header
geometry_msgs/PolygonStamped[] polygon
int32[] labels
float32[] likelihood
string[] names
int64[] indices
